Objects/object.c

```diff
@@ -2590,6 +2590,20 @@ PyUnstable_Object_EnableDeferredRefcount(PyObject *op)
 #endif
 }
 
+int
+PyUnstable_TryIncRef(PyObject *op)
+{
+    return _Py_TryIncref(op);
+}
+
+void
```

Comment:
According to the documentation, it should return int type :)
``c:function:: int PyUnstable_EnableTryIncRef(PyObject *obj)``

Reply:
I changed the return type in the docs.